charts/openmetadata/templates/service.yaml

```diff
@@ -3,11 +3,11 @@ kind: Service
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.service.annotations }}
+    {{- include "OpenMetadata.labels" . | nindent 4 }} 
   annotations:
+    {{- with .Values.service.annotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
{{- include "OpenMetadata.annotations" . | nindent 4 }}.

Refactor to _helper.tpl as mentioned above in review.

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}